### ISSUE TITLE
VictoriaMetrics

### ISSUE THREAD

**Issue by Antirecord** [banzaicloud/spark-metrics#83]
Good afternoon!  if we don't have pushgateway, is it possible to send metrics to VictoriaMetrics using your library?

**Comment by stoader** [comment]
I'm not familiar with VictoriaMetrics. If the rest API of VictoriaMetrics is 100% compatible with [Pushgateway's HTTP API](https://github.com/prometheus/pushgateway#api) in theory it should work.

**Comment by g1thubhub** [comment]
Yes it works, I had similar questions and have implemented a push and pull-based approach based on VictoriaMetrics in this project:  https://github.com/xonai-computing/xonai-dashboard
